src/plugins/axes/AxesConfig.qml

```diff
@@ -0,0 +1,259 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 2.1
+import QtQuick.Dialogs 1.0
+import QtQuick.Layouts 1.3
+import "qrc:/qml"
+
+GridLayout {
+  columns: 4
+  columnSpacing: 10
+  Layout.minimumWidth: 200
+  Layout.minimumHeight: 500
+  anchors.fill: parent
+  anchors.leftMargin: 10
+  anchors.rightMargin: 10
+
+  // Left spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  ComboBox {
+    Layout.alignment: Qt.AlignHCenter
+    Layout.columnSpan: 2
+    objectName: "ComboBoxEntities"
+    model: AxesConfig.comboList
+    onActivated: {
+      AxesConfig.onCurrentIndexChanged(index)
+      axesLength.value = AxesConfig.length
+      x.value = AxesConfig.axesX
+      y.value = AxesConfig.axesY
+      z.value = AxesConfig.axesZ
+      roll.value = AxesConfig.axesRoll
+      pitch.value = AxesConfig.axesPitch
+      yaw.value = AxesConfig.axesYaw
+      showAxes.checkState = AxesConfig.showAxes
+      isArrow.checkState = AxesConfig.showArrow
+    }
+    onAccepted: {
+         if (editableCombo.find(currentText) === -1) {
+             model.append({text: editText})
+             currentIndex = editableCombo.find(editText)
+         }
+     }
+  }
+
+  // Right spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  // Left spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  CheckBox {
+    id: showAxes
+    Layout.alignment: Qt.AlignHCenter
+    Layout.columnSpan: 2
+    text: qsTr("Show Axes")
+    checked: true
+    onClicked: {
+      AxesConfig.OnShow(checked)
+    }
+  }
+
+  // Right spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  // Left spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  CheckBox {
+    id: isArrow
+    Layout.alignment: Qt.AlignHCenter
+    Layout.columnSpan: 2
+    text: qsTr("Show arrows")
+    checked: true
+    onClicked: {
+      AxesConfig.OnTypeAxes(checked)
+    }
+  }
+
+  // Right spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  // Left spacer
+  Item {
+    Layout.columnSpan: 1
+    Layout.rowSpan: 1
+    Layout.fillWidth: true
+  }
+
+  Text {
+    Layout.columnSpan: 1
+    color: "dimgrey"
+    text: "Size(meters):"
+    Layout.fillWidth: true
+  }
+
+  IgnSpinBox {
+    Layout.columnSpan: 1
+    id: axesLength
+    maximumValue: 100
+    minimumValue: 1
+    value: 1
+    decimals: 2
+    stepSize: 0.5
+    onEditingFinished: AxesConfig.UpdateLength(axesLength.value)
```

Comment:
Changing the size doesn't seem to work for me. I change the number, press enter, and the old number comes back.

Reply:
I checked this and it's working for me

![axesconfig](https://user-images.githubusercontent.com/1933907/115054136-2bd25680-9ee0-11eb-8b36-ecb5fcd9649c.gif)

Reply:
It works for me now!